modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_4/DDMStructureLinkDLFileEntryTypeUpgradeProcess.java

```diff
@@ -88,43 +85,56 @@ private void _fixDataDefinitionIdFields() throws Exception {
 
 				try (ResultSet resultSet2 = preparedStatement2.executeQuery()) {
 					if (resultSet2.next()) {
-						preparedStatement3.setLong(1, resultSet2.getLong(1));
-						preparedStatement3.setLong(2, fileEntryTypeId);
-
-						preparedStatement3.addBatch();
+						long structureId = resultSet2.getLong(1);
```

Comment:
This variable is not necessary. Could you please make it inline?

Thx.

Reply:
Hi @rodrigopaulino 

The variable is used in a lambda, if you inline it will be asked to be surrounded by a try/catch because the _resultSet.getLong_ throws an _Unhandled exception: java.sql.SQLException_ so it needs to be declared outside the lambda.

Reply:
Thanks for clarifying, @AliciaGarciaGarcia. We can move this forward then.